lib/python/src/bailo/helper/schema.py

```diff
@@ -107,3 +107,14 @@ def __unpack(self, res) -> None:
             self.kind = SchemaKind.ACCESS_REQUEST
 
         logger.info(f"Attributes for Schema ID %s successfully unpacked.", self.schema_id)
+
+    @classmethod
```

Comment:
I think this could actually be a `staticmethod` instead since we aren't using `cls` unless we want to create a list of `Schema()` objects. If we just want the IDs then `staticmethod` is best